src/library/Book.java

```diff
@@ -0,0 +1,68 @@
+package library;
+
+
+import java.util.Objects;
+
+public class Book implements Comparable<Book> {
+
+    private int id;
+    private String title;
+    private String genre;
+
+    public Book(int id, String title, String genre) {
+        this.id = id;
+        this.title = title;
+        this.genre = genre;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getGenre() {
+        return genre;
+    }
+
+    public void setGenre(String genre) {
+        this.genre = genre;
+    }
+
+    @Override
+    public String toString() {
+        return "Book{" +
+                "id=" + id +
+                ", title='" + title + '\'' +
+                ", genre='" + genre + '\'' +
+                '}';
+    }
+
+    @Override
+    public int compareTo(Book o) {
+        return this.getTitle().toLowerCase().compareTo(o.getTitle().toLowerCase());
+    }
```

Comment:
Let's keep comparing strategy not hardcoded and depending on users chose. remove it

Reply:
Done. Comparable and method compareTo were removed. Comparator was used.